src/style-dictionary-dist/TerrawareTheme.ts

```diff
@@ -369,193 +371,188 @@ type ThemeDictionary = {
+    'TwClrBgBrandGhostActive': 'rgb(44 134 88 / 20%)',
+    'TwClrBgBrandGhostHover': 'rgb(44 134 88 / 10%)',
+    'TwClrBgBrandHover': 'linear-gradient(rgb(35 106 70 / 50%), rgb(35 106 70 / 50%)), linear-gradient(#2C8658, #2C8658)',
+    'TwClrBgBrandTertiary': '#E2F6EC',
+    'TwClrBrdrBrand': '#2C8658',
+    'TwClrIcnBrand': '#2C8658',
+    'TwClrIcnOnBrand': '#FFFFFF',
+    'TwClrTxtBrand': '#2C8658',
+    'TwClrTxtOnBrand': '#FFFFFF',
+    'TwClrBgDanger': '#CB4D45',
+    'TwClrBgDangerActive': '#A8362F',
+    'TwClrBgDangerGhost': 'rgb(203 77 69 / 0%)',
+    'TwClrBgDangerGhostActive': 'rgb(203 77 69 / 20%)',
+    'TwClrBgDangerGhostHover': 'rgb(203 77 69 / 10%)',
+    'TwClrBgDangerHover': 'linear-gradient(rgb(168 54 47 / 50%), rgb(168 54 47 / 50%)), linear-gradient(#CB4D45, #CB4D45)',
+    'TwClrBgDangerTertiary': '#FAEEED',
+    'TwClrBrdrDanger': '#CB4D45',
+    'TwClrBrdrDangerActive': '#A8362F',
+    'TwClrBrdrDangerHover': 'linear-gradient(rgb(168 54 47 / 50%), rgb(168 54 47 / 50%)), linear-gradient(#CB4D45, #CB4D45)',
+    'TwClrIcnDanger': '#CB4D45',
+    'TwClrIcnOnDanger': '#FFFFFF',
+    'TwClrTxtDanger': '#CB4D45',
+    'TwClrTxtOnDanger': '#FFFFFF',
+    'TwClrBgInfo': '#7F775B',
+    'TwClrBgInfoTertiary': '#F1F0EC',
+    'TwClrBrdrInfo': '#7F775B',
+    'TwClrIcnInfo': '#7F775B',
+    'TwClrIcnOnInfo': '#FFFFFF',
+    'TwClrTxtInfo': '#645E48',
+    'TwClrTxtOnInfo': '#FFFFFF',
+    'TwClrBgSelected': '#6172BE',
+    'TwClrBgSelectedTertiary': '#EFF1F8',
+    'TwClrBgSelectedGhost': 'rgb(97 114 190 / 0%)',
+    'TwClrBgSelectedGhostActive': 'rgb(97 114 190 / 20%)',
+    'TwClrBgSelectedGhostHover': 'rgb(97 114 190 / 10%)',
+    'TwClrBrdrSelected': '#6172BE',
+    'TwClrIcnOnSelected': '#FFFFFF',
+    'TwClrIcnSelected': '#6172BE',
+    'TwClrTxtOnSelected': '#FFFFFF',
+    'TwClrTxtSelected': '#6172BE',
+    'TwClrBgSuccess': '#5D822B',
+    'TwClrBgSuccessTertiary': '#EAF3DC',
+    'TwClrBgSuccessActive': '#496622',
+    'TwClrBgSuccessHover': 'linear-gradient(rgb(73 102 34 / 50%), rgb(73 102 34 / 50%)), linear-gradient(#5D822B, #5D822B)',
+    'TwClrBrdrSuccess': '#5D822B',
+    'TwClrIcnOnSuccess': '#FFFFFF',
+    'TwClrIcnSuccess': '#5D822B',
+    'TwClrTxtOnSuccess': '#FFFFFF',
+    'TwClrTxtSuccess': '#5D822B',
+    'TwClrBgWarning': '#A36A2D',
+    'TwClrBgWarningActive': '#815424',
+    'TwClrBgWarningHover': 'linear-gradient(rgb(129 84 36 / 50%), rgb(129 84 36 / 50%)), linear-gradient(#A36A2D, #A36A2D)',
+    'TwClrBgWarningTertiary': '#F8EFE5',
+    'TwClrBrdrWarning': '#A36A2D',
+    'TwClrBrdrWarningActive': '#815424',
+    'TwClrBrdrWarningHover': 'linear-gradient(rgb(129 84 36 / 50%), rgb(129 84 36 / 50%)), linear-gradient(#A36A2D, #A36A2D)',
+    'TwClrIcnOnWarning': '#FFFFFF',
+    'TwClrIcnWarning': '#A36A2D',
+    'TwClrTxtOnWarning': '#FFFFFF',
+    'TwClrTxtWarning': '#A36A2D',
+    'TwClrBgAccent': '#EF7047',
+    'TwClrTxtAccent': '#EF7047',
+    'TwClrTxtOnAccent': '#FFFFFF'
+  }
```

Comment:
Do you know which colors were removed?

Reply:
I don't think any colors were removed.  If you're just looking at line numbers then there were some unused lines at the top of the file that were removed because they were causing ESLint to trip

Reply:
I don't know.  Since it's autogenerated from the style dictionary I suspect it was design team removing unused colors.  I can double-check with Chudo.